submariner/multiple-cable-drivers.md

```diff
@@ -0,0 +1,192 @@
+# Support fine-grained cluster connectivity and multiple cable drivers
```

Comment:
On the one hand, this could be separated into two proposals. But they share a common foundation which is defining the attributes of cluster adjacencies. My suggestion would actually be to align them more closely and use the same policy CRD for defining adjacencies and their cable properties.

Reply:
I think there was an original iteration where they were combined but then we decided to separate them. There are different use cases for the what vs the how.